package.json

```diff
@@ -47,8 +47,10 @@
     "webpack-merge": "^4.1.1"
   },
   "dependencies": {
+    "antd": "^3.16.3",
     "babel-polyfill": "^6.26.0",
     "livereactload": "^4.0.0-beta.2",
+    "lodash": "^4.17.11",
```

Comment:
We use a large subset of lodash functions? If we only use lodash.get, import only this